### DISCUSSION TITLE
trivy 0.49.1 scanning error

### DISCUSSION THREAD

**Discussion by trinishanmukha** [aquasecurity/trivy#6096] [Bugs]
### Description

Using new trivy version to scan java container resulting in below error
image scan error: scan error: scan failed: failed analysis: analyze error: pipeline error: failed to analyze layer : post analysis error: post analysis error: Unable to initialize the Java DB: Java DB update failed: DB download error: oci download error: copy error: http2: server sent GOAWAY and closed the connection; LastStreamID=1, ErrCode=NO_ERROR, debug="graceful shutdown"

can get a temporary workaround by adding --security-checks vuln flag

### Desired Behavior

Error shouldnot happen even without the flag

### Actual Behavior

Getting error

### Reproduction Steps

```bash
1.
2.
3.
...
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
image scan error: scan error: scan failed: failed analysis: analyze error: pipeline error: failed to analyze layer : post analysis error: post analysis error: Unable to initialize the Java DB: Java DB update failed: DB download error: oci download error: copy error: http2: server sent GOAWAY and closed the connection; LastStreamID=1, ErrCode=NO_ERROR, debug="graceful shutdown"
```

### Operating System

linux

### Version

```bash
latest
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
Hello @trinishanmukha 
Thanks for your report!

This is strange case. Looks like there is some problem when Trivy tries to download `trivy-java-db`. 
Also, this should not be affected by the `--security-checks vuln` flag.

Do you consistently see this error?
Did Trivy download `trivy-java-db` with `--security-checks vuln` flag?
